include/xbee/ble_frames.h

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2024 Digi International Inc.,
+ * All rights not expressly granted are reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * Digi International Inc., 9350 Excelsior Blvd., Suite 700, Hopkins, MN 55343
+ * ===========================================================================
+ */
+
+/**
+    @addtogroup xbee_ble
+    @{
+    @file xbee/ble_frames.h
+
+    Frame definitions and support functions for BLE frames
+    (0x34, 0xB4, 0xB5 and 0xB7).
+
+    For use with the XBee BLU
+*/
+
+#ifndef XBEE_BLE_FRAMES_H
+#define XBEE_BLE_FRAMES_H
+
+#include "xbee/platform.h"
+
+XBEE_BEGIN_DECLS
+
+/// Flag used in advertisement responses, in the 0xB4 and 0xB7 frame types
+#define XBEE_BLE_ADVERT_FLAG_CONNECTABLE  ( 1 << 0 ) // Advertisement is connectable
+
+// Max and min values for the scan window and the scan interval in microseconds.
+#define XBEE_BLE_SCAN_VALUES_MAX_USEC       (0x270FD8F)
+#define XBEE_BLE_SCAN_VALUES_MIN_USEC       (0x9C4)
```

Comment:
The upper limit here is `0xFFFF * 0.625` milliseconds, and is the limit exposed by the SiLabs Gecko SDK Bluetooth stack (and thus, the XBee's limit).

I agree on the formatting of `2500` being better than `0x9C4`. (Not sure why some folks within Digi like to write things like this in the less-human-readable form, other than maybe because it helps when looking at a hex dump of frames?)

Reply:
I think the hex format is due to default XCTU settings.  Even worse, XCTU doesn't use a `0x` prefix which led to some confusion in the past.